### PR TITLE
LPS-161245 Creates a new property called resizeDirection for liferay-editor:editor tag

### PR DESCRIPTION
**Pull request by diegonvs**
Came from: https://github.com/brianchandotcom/liferay-portal/pull/122649#issuecomment-1237515612

I tested locally and it's working.

## How to test locally:

1. Go to *ckeditor-sample-web module
2. Inside liferay-editor:editor tag call, replace with the following snippet:

```jsp
<liferay-editor:editor
	contents="<%= sampleEditorContents %>"
	editorName="ckeditor"
	name="sampleBalloonEditor"
	placeholder="content"
	resizable="<%= Boolean.TRUE %>"
	resizeDirection="vertical"
/>
```
3. Drop a CKEditor Sample on a page and try to resize. You'll notice we can only resize it vertically.

### CODE REVIEW

**Comment by diegonvs** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/fd6306d87c...a111b749a1

**Comment by brianchandotcom** [comment]
@diegonvs fyi a111b749a1eb157a0f0e4131f327ddaac6e00fd0

**Comment by diegonvs** [comment]
Thanks for adding https://github.com/brianchandotcom/liferay-portal/commit/a111b749a1eb157a0f0e4131f327ddaac6e00fd0.

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java [hunk 1]
```diff
@@ -113,7 +113,12 @@ public void populateConfigJSONObject(
 				CKEditorConstants.ATTRIBUTE_NAMESPACE + ":resizable"));
 
 		if (resizable) {
-			jsonObject.put("resize_dir", "vertical");
+			String resizeDirection = GetterUtil.getString(
+				inputEditorTaglibAttributes.get(
+					CKEditorConstants.ATTRIBUTE_NAMESPACE +
+						":resizeDirection"));
+
+			jsonObject.put("resizeDirection", resizeDirection);
```
Comment (dsanz) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java:121]: shouldn't this one still be `resize_dir`?

Comment (diegonvs) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java:121]: You're correct. https://github.com/brianchandotcom/liferay-portal/pull/123119

**Pull request closed without merging**